src/pokemon.c

```diff
@@ -4388,6 +4388,27 @@ u16 GetEvolutionTargetSpecies(struct Pokemon *mon, u8 mode, u16 evolutionItem, s
             }
         }
         break;
+    case EVO_MODE_CANT_STOP:
```

Comment:
Doesn't `EVO_MODE_TRADE` do the same thing? So you could rename if to `EVO_MODE_CANT_STOP` and include `EVO_LEVEL_ITEM_COUNT_999` there.

Reply:
EVO_MODE_TRADE is only called in places that have to do with TradeEvolutionScene, which is not the same thing.